pkg/limactlutil/limactlutil.go

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: Copyright The Lima Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package limactlutil
+
+import (
+	"bytes"
+	"cmp"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/lima-vm/lima/v2/pkg/limatype"
+)
+
+// Path returns the path to the `limactl` executable.
+func Path() (string, error) {
+	limactl := cmp.Or(os.Getenv("LIMACTL"), "limactl")
```

Comment:
Why is this needed? It needs to be documented, if we do need it. And then it should be a pattern for all plugins.

Right now we go to extra effort that the version of `limactl` that was used to invoke the plugin is first on the `PATH` before the plugin is started.

If this is purely for development/testing, then the variable should start with an underscore.